src/renderer/components/wallet/txs/send/SendFormBCH.tsx

```diff
@@ -32,13 +31,13 @@ import { WalletBalance } from '../../../../services/wallet/types'
 import { PasswordModal } from '../../../modal/password'
 import * as StyledR from '../../../shared/form/Radio.styles'
 import { MaxBalanceButton } from '../../../uielements/button/MaxBalanceButton'
-import { WalletTypeLabel } from '../../../uielements/common/Common.styles'
 import { UIFeesRD } from '../../../uielements/fees'
 import { Input, InputBigNumber } from '../../../uielements/input'
 import { AccountSelector } from '../../account'
 import * as Styled from '../TxForm.styles'
 import { validateTxAmountInput } from '../TxForm.util'
 import { DEFAULT_FEE_OPTION } from './Send.const'
+import { getMatchedWalletType, getRenderedWalletType } from './Send.helpers'
```

Comment:
```
import * as H from './Send.helpers'
```

to use namespace `N` to avoid collision of same function names, e.g `H.matchedWalletType`